goggies/src/App.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import './App.css';
+import remote from './js/Service'
+import Controller from './components/GoggieList'
+
+export default class App extends React.Component {
+  constructor(props) {
+    super(props)
+    this.fetcher=new remote();
+    this.state={
+      goggiesList:["Loading"]
+    }
```

Comment:
Interesting way to account for the loading state since this is a place holder while we wait on a response from the API. I like to use a flag in the state determining if we render a loading gif or video, until the `goggiesList` is populated.